Vic2ToHoI4/Data_Files/configurables/regions.txt

```diff
@@ -1,1489 +1,2252 @@
-﻿central_asia = {
-	name = "Central Asia"
-	adjective = "Central Asian"
+# "Name" and "Adjective" refer to the slave name!
+# Provinces are HoI4 ones, arranged by Vic2 states
+# A dominion, to work, needs to have the corresponding
+# <region>_emblem.tga file in the flags folder
+# Freed name and adjective yet to be implemented
+
+canada = {
+	name = "$canada$"
```

Comment:
1. I thought HoI4 supported nested locs, I will test on my machine to see how it loads
2. Couldn't we use the region code name instead of the localized name in filename generation? E.g. *British southern_africa* instead of *British $south_africa$*/*British South Africa*?

Reply:
I figured it out and I should have remembered from the beginning: you need to add <region> and <region_ADJ> to one of the .csv files under Data_Files\configurables\Vic2Localisations

Reply:
Done